### ISSUE TITLE
erleans_stream_manager crashes w/ unset variable. 

### ISSUE THREAD

**Issue by Licenser** [SpaceTime-IoT/erleans#11]
Starting erleans fails with (probably?) some unset variable:

```
2> application:ensure_all_started(erleans).
%%% ...
22:01:35.306 [error] gen_server erleans_stream_manager terminated with reason: no match of right hand value undefined in erleans_config:get/1 line 29
```

**Comment by Licenser** [comment]
Env (default):

```
11> application:get_all_env(erleans).
[{included_applications,[]},
 {default_eval_interval,60},
 {default_lease_time,2700000},
 {default_life_time,0}]
```

**Comment by tsloughter** [comment]
Ah, yup. Will simply not start the manager if there are no configured stream providers for now.

**Comment by Licenser** [comment]
any hint what to variable is looked for so I can temporarily just set it? I tried looking in `erleans_stream_manager` but there sees no erlans_config call in there.

**Comment by tsloughter** [comment]
Actually, it is two fold. It wants to check the persistent store for any saved streams, so it will try to find the provider for erleans_streams (which if not set directly is `default_provider`. But if you set that you need the module too. So currently the easiest way to play around without adding a provider is:

```
rebar3 as test shell --config=test/sys.config
```

This will use the CT tests config and module `ets_provider`.

**Comment by tsloughter** [comment]
Resolved in #27
